File: Trabajo3/CodigoBusquedaLineal.cpp
#include <iostream>
using namespace std;

int main() {
    int arreglo[] = {12, 45, 23, 67, 89, 34};
    int tamaño = sizeof(arreglo) / sizeof(arreglo[0]);
    int valorBuscado = 67;
    bool encontrado = false;

    for (int i = 0; i < tamaño; i++) {
        if (arreglo[i] == valorBuscado) {
            cout << "Valor encontrado en el índice " << i << endl;
            encontrado = true;
            break;
        }
    }

    if (!encontrado) {
        cout << "Valor no encontrado en el arreglo." << endl;
    }

    return 0;
}

File: Trabajo3/CodigoBusquedaLIneal.java
public class Main {
    public static void main(String[] args) {
        int[] arreglo = {12,23,57,45,34, 67, 89, 34};
        int valorBuscado = 34;
        boolean encontrado = false;

        for (int i = 0; i < arreglo.length; i++) {
            if (arreglo[i] == valorBuscado) {
                System.out.println("Valor encontrado en el índice " + i);
                encontrado = true;
                break;
            }
        }

        if (!encontrado) {
            System.out.println("Valor no encontrado en el arreglo.");
        }
    }
}